deploy/examples/toolbox-rook-image.yaml

```diff
@@ -0,0 +1,136 @@
+#################################################################################################################
```

Comment:
It would be good to have some integration test just to know that this toolbox pod could start. Perhaps in one of the tests in canary-integration-test.yaml you could just create this yaml and check that the pod gets to running state.